x-pack/platform/plugins/shared/alerting/server/task_runner/task_runner.ts

```diff
@@ -679,119 +670,50 @@ export class TaskRunner<
-          const errorSourceTag = `${errorSource}-error`;
-
-          if (isAlertSavedObjectNotFoundError(err, ruleId) || isClusterBlockError(err)) {
-            const message = `Executing Rule ${spaceId}:${
-              this.ruleType.id
-            }:${ruleId} has resulted in Error: ${getEsErrorMessage(err)}`;
-            this.logger.debug(message, {
-              tags: [this.ruleType.id, ruleId, 'rule-run-failed', errorSourceTag],
-            });
-          } else {
-            const error = this.stackTraceLog ? this.stackTraceLog.message : err;
-            const stack = this.stackTraceLog ? this.stackTraceLog.stackTrace : err.stack;
-            const message = `Executing Rule ${spaceId}:${
-              this.ruleType.id
-            }:${ruleId} has resulted in Error: ${getEsErrorMessage(error)} - ${stack ?? ''}`;
-            this.logger.error(message, {
-              tags: [this.ruleType.id, ruleId, 'rule-run-failed', errorSourceTag],
-              error: { stack_trace: stack },
-            });
-          }
-          return originalState;
-        }
-      ),
-      schedule: resolveErr<IntervalSchedule | undefined, Error>(schedule, (error) => {
-        if (isAlertSavedObjectNotFoundError(error, ruleId)) {
-          const spaceMessage = spaceId ? `in the "${spaceId}" space ` : '';
-          this.logger.warn(
-            `Unable to execute rule "${ruleId}" ${spaceMessage}because ${error.message} - this rule will not be rescheduled. To restart rule execution, try disabling and re-enabling this rule.`
-          );
-          throwUnrecoverableError(error);
-        }
-
-        let retryInterval = taskSchedule?.interval ?? FALLBACK_RETRY_INTERVAL;
-
-        // Set retry interval smaller for ES connectivity errors
-        if (isEsUnavailableError(error, ruleId)) {
-          retryInterval =
-            parseDuration(retryInterval) > parseDuration(CONNECTIVITY_RETRY_INTERVAL)
-              ? CONNECTIVITY_RETRY_INTERVAL
-              : retryInterval;
-        }
-
-        if (isClusterBlockError(error)) {
-          retryInterval = CLUSTER_BLOCKED_EXCEPTION_RETRY_INTERVAL;
-        }
-
-        return { interval: retryInterval };
+      state: getState({
+        runRuleResult,
+        startedAt,
+        ruleId,
+        spaceId,
+        ruleTypeId: this.ruleType.id,
+        logger: this.logger,
+        stackTraceLog: this.stackTraceLog,
+        originalState,
+      }),
+      schedule: getSchedule({
+        schedule,
+        ruleId,
+        spaceId,
+        retryInterval: taskSchedule?.interval ?? FALLBACK_RETRY_INTERVAL,
+        logger: this.logger,
+      }),
+      ...getTaskRunError({
+        runRuleResult,
+        logger: this.logger,
+        ruleResult: this.ruleResult,
+        ruleTypeId: this.ruleType.id,
+        ruleId,
       }),
-      monitoring: this.ruleMonitoring.getMonitoring(),
-      ...getTaskRunError(stateWithMetrics),
```

Comment:
I removed `monitoring` from the alerting TaskRunner result as TaskManager doesn't expect and use it.